src/Azure/ServiceBus/ReceiveMode.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace HalloVerden\AzureServiceBusMessengerBundle\Azure\ServiceBus;
+
+use Symfony\Component\HttpFoundation\Request;
+
+enum ReceiveMode {
+  case RECEIVE_AND_DELETE;
+  case PEEK_LOCK;
+
+  /**
+   * @return string
+   */
+  public function getMethod(): string {
```

Comment:
Would suggest being more specific about method (i.e. `getHttpMethod`), in case the enum becomes relevant for other ways of communicating with Azure in the future.